modules/archetypes/lib/policy_assignments/policy_assignment_es_deny_public_ip.tmpl.json

```diff
@@ -16,17 +23,11 @@
         "value": "Deny"
       }
     },
-    "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/6c112d4e-5bc7-47ae-a041-ea2d9dccd749",
-    "nonComplianceMessages": [
-      {
-        "message": "Public IPs {enforcementMode} not be created in this scope."
-      }
-    ],
     "scope": "${current_scope_resource_id}",
-    "enforcementMode": "Default"
```

Comment:
Why is this set to Default?

Reply:
@matt-FFFFFF and @jtracey93 I am not sure if this was by design. It does look like it would override the setting in the code. Any thoughts?